src/events/message.rs

```diff
@@ -47,6 +54,50 @@ pub async fn message(ctx: client::Context, msg: Message) -> Result<()> {
     Ok(())
 }
 
+async fn handle_highlighting(ctx: &client::Context, msg: &Message) -> Result<()> {
+    let db = ctx.get_db().await;
+    let highlights = db.get_highlights().await?;
+
+    for i in &mut msg
+        .content
+        .split_whitespace()
+        .into_iter()
+        .map(|p| p.to_string())
+    {
+        // checks if the database contains our word
+        if let Some(m) = highlights.get(&i.to_string()) {
+            // iterates over each user that subscribed to that highlight and send them a message 
+            for x in m {
+                let channel = msg.channel(&ctx).await.unwrap().guild().unwrap().name;
+                let guild = msg.guild(&ctx).await.unwrap().name;
```

Comment:
plox don't unwrap REEEEEEEEEEEEEEEEE

Reply:
It's a NoneError, I assumed that it was only an error because of syntactic reasons.